xbmc/guilib/GUIAction.h

```diff
@@ -21,21 +21,32 @@ class CGUIListItem; typedef std::shared_ptr<CGUIListItem> CGUIListItemPtr;
 class CGUIAction
 {
 public:
+  /**
+   * Pair containing the info condition (as string) and the action (as string)
+   */
+  struct CondActionPair
```

Comment:
I would not name the type `Pair`. If we decide later to add attribute(s), we have to rename the type. Naming the type `Pair`somehow exposes an implementation detail.